content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx

```diff
@@ -0,0 +1,321 @@
+---
+title: 'Upgrade to Prisma 5'
+metaTitle: 'Upgrade to Prisma 5'
+metaDescription: 'Guides on how to upgrade to Prisma 5'
+tocDepth: 3
+toc: true
+---
+
+<TopBlock>
+
+Prisma 5.0.0 introduces a number of **breaking changes**. This guide explains how upgrading might affect your application and gives instructions on how to handle any breaking changes.
+
+</TopBlock>
+
+## Major changes
+
+This section gives an overview of major breaking changes in Prisma 5.
+
+We recommend that you first address any Prisma schema validation errors, then pull your database to reflect new Prisma schema capabilities, and finally fix any type errors in Prisma Client and validate by running your test suite.
+
+### Node.js minimum version change
+
+From Prism version 5.0.0, the minimum version of Node.js supported is 16.13.0. If your project uses an earlier version of Node.js, you will need to update it.
+
+See our [system requirements](/reference/system-requirements) for all minimum version requirements.
+
+<Admonition type="info">
+
+Node.js v16.x is reaching [end-of-life on 11 September, 2023](https://nodejs.org/en/blog/announcements/nodejs16-eol) in order to coincide with the end-of-life of OpenSSL 1.1.1. For that reason we recommend upgrading to Node.js v18.16.1.
+
+Please note that Prisma 5 will be the last major version of Prisma to support Node.js v16.
+
+</Admonition>
+
+### PostgreSQL minimum version change
+
+From Prisma version 5.0.0, the minimum version of PostgreSQL supported is version 9.6. If your project uses an earlier version of PostgreSQL, you will need to update it.
+
+Our tests have not been testing PostgreSQL 9.4 and 9.5 for some time, so we are formally removing support.
+
+### Prisma Client embedded SQLite version updated
+
+With Prisma version 5.0.0, we have upgraded the embedded version of SQLite from `3.35.4` to `3.41.2`. We did not see any breaking changes and don't anticipate any changes needed in user projects, but if you are using SQLite, make sure to check [the SQLite changelog](https://www.sqlite.org/changes.html)
+
+### Removal of array shortcuts
+
+Prisma version 5.0.0 drops support for array shortcuts, which were a way to omit a wrapping array for single values.
+
+#### `OR` clauses
+
+The following code in Prisma 4:
```

Comment:
This also could be code from any other previous version. 
(Also true for all other examples)